n3fit/runcards/examples/nnpdf40-like_pos_cut.yaml

```
@@ -0,0 +1,189 @@
+datacuts:
+  t0pdfset: 240329-01-rs-nnpdf40like-baseline
+  q2min: 3.49
+  w2min: 12.5
+
+############################################################
+theory:
+  theoryid: 708
+sampling:
+  separate_multiplicative: false
+
+############################################################
+trvlseed: 591866982
+nnseed: 945709987
+mcseed: 519562661
+genrep: true
+
+parameters: # This defines the parameter dictionary that is passed to the Model Trainer
+  nodes_per_layer: [25, 20, 8]
+  activation_per_layer: [tanh, tanh, linear]
+  initializer: glorot_normal
+  optimizer:
+    clipnorm: 6.073e-6
+    learning_rate: 2.621e-3
+    optimizer_name: Nadam
+  epochs: 17000
+  positivity:
+    initial: 184.8
+    multiplier:
+  integrability:
+    initial: 10
+    multiplier:
+  stopping_patience: 0.1
+  layer_type: dense
+  dropout: 0.0
+  threshold_chi2: 3.5
+
+fitting:
+  fitbasis: EVOL
+  basis:
+  - {fl: sng, trainable: false, smallx: [1.092, 1.118], largex: [1.479, 3.037]}
+  - {fl: g, trainable: false, smallx: [0.8048, 1.053], largex: [2.828, 5.503]}
+  - {fl: v, trainable: false, smallx: [0.4678, 0.744], largex: [1.547, 3.319]}
+  - {fl: v3, trainable: false, smallx: [0.1268, 0.4549], largex: [1.725, 3.38]}
+  - {fl: v8, trainable: false, smallx: [0.5864, 0.7822], largex: [1.533, 3.282]}
+  - {fl: t3, trainable: false, smallx: [-0.436, 1.0], largex: [1.771, 3.47]}
+  - {fl: t8, trainable: false, smallx: [0.5878, 0.874], largex: [1.548, 3.265]}
+  - {fl: t15, trainable: false, smallx: [1.087, 1.146], largex: [1.493, 3.441]}
+
+############################################################
+positivity:
+  posdatasets:
+  - {dataset: NNPDF_POS_2P24GEV_F2U, maxlambda: 1e6}        # Positivity Lagrange Multiplier
+  - {dataset: NNPDF_POS_2P24GEV_F2D, maxlambda: 1e6}
+  - {dataset: NNPDF_POS_2P24GEV_F2S, maxlambda: 1e6}
+  - {dataset: NNPDF_POS_2P24GEV_FLL-19PTS, maxlambda: 1e6}
+  - {dataset: NNPDF_POS_2P24GEV_DYU, maxlambda: 1e10}
+  - {dataset: NNPDF_POS_2P24GEV_DYD, maxlambda: 1e10}
+  - {dataset: NNPDF_POS_2P24GEV_DYS, maxlambda: 1e10}
+  - {dataset: NNPDF_POS_2P24GEV_F2C-17PTS, maxlambda: 1e6}
+  - {dataset: NNPDF_POS_2P24GEV_XUQ, maxlambda: 1e6}        # Positivity of MSbar PDFs
+  - {dataset: NNPDF_POS_2P24GEV_XUB, maxlambda: 1e6}
+  - {dataset: NNPDF_POS_2P24GEV_XDQ, maxlambda: 1e6}
+  - {dataset: NNPDF_POS_2P24GEV_XDB, maxlambda: 1e6}
+  - {dataset: NNPDF_POS_2P24GEV_XSQ, maxlambda: 1e6}
+  - {dataset: NNPDF_POS_2P24GEV_XSB, maxlambda: 1e6}
+  - {dataset: NNPDF_POS_2P24GEV_XGL, maxlambda: 1e6}
+
+added_filter_rules:
+    - dataset: NNPDF_POS_2P24GEV_XGL
+      rule: "x > 0.1"
+
+    - dataset: NNPDF_POS_2P24GEV_XUQ
+      rule: "x > 0.1"
+
+    - dataset: NNPDF_POS_2P24GEV_XUB
+      rule: "x > 0.1"
+
+    - dataset: NNPDF_POS_2P24GEV_XDQ
+      rule: "x > 0.1"
+
+    - dataset: NNPDF_POS_2P24GEV_XDB
+      rule: "x > 0.1"
+
+    - dataset: NNPDF_POS_2P24GEV_XSQ
+      rule: "x > 0.1"
+
+    - dataset: NNPDF_POS_2P24GEV_XSB
```

Given that you are setting the `POS_XPDF` process type, you should be able to also do 

```suggestion
    - process_type: POS_XPDF
```

and have one single cut.